profile/flatpak.fish

```diff
@@ -0,0 +1,17 @@
+if type -q flatpak
+    # Set XDG_DATA_DIRS to include Flatpak installations
+
+    set -x --path XDG_DATA_DIRS
```

Comment:
@alexlarsson I think calling set this way deletes the variable. I am not sure if this is what you wanted to do?

When I use this code to fix my flatpak 1.12.x setup with fish, it deletes a path put into XDG_DATA_DIRS by KDE.

Reply:
If this profile script is wrong, please [open an issue](https://github.com/flatpak/flatpak/issues), or preferably [open a pull request](https://github.com/flatpak/flatpak/pulls) that gives it the correct behaviour, rather than commenting on a closed pull request.

The maintainers of Flatpak do not use fish, so we don't know what is correct here. If you use fish, you are in a better position to know what is correct here than we are.

(Also, @alexlarsson didn't write this code.)

Reply:
For whatever its worth, I do use `fish` and `flatpak` completion has seemed to work fine for me.